src/bootstrap/src/core/builder/cargo.rs

```diff
@@ -114,7 +114,7 @@ impl Cargo {
         match cmd_kind {
             // No need to configure the target linker for these command types,
             // as they don't invoke rustc at all.
```

Comment:
Could you update this comment since we're now invoking rustc?

Reply:
Good point